src/DynamoCoreWpf/ViewModels/Watch3D/DefaultWatch3DViewModel.cs

```diff
@@ -562,9 +562,9 @@ private void ToggleCanNavigateBackground(object parameter)
             InstrumentationLogger.LogAnonymousScreen(CanNavigateBackground ? "Geometry" : "Nodes");
         }
 
-        private bool CanToggleCanNavigateBackground(object parameter)
+        protected virtual bool CanToggleCanNavigateBackground(object parameter)
         {
-            return true;
+            return false;
```

Comment:
In the `DefaultWatch3DViewModel` we should always disallow toggling to navigate background.